{{cookiecutter.project_slug}}/clients/web/vue3/.env.local.example

```diff
@@ -1,5 +1,5 @@
 # When running the backend manually locally with `python manage.py runserver`
-# VITE_DEV_BACKEND_URL="http://localhost:8000"
+# VITE_DEV_BACKEND_URL="http://127.0.0.1:8000"
```

Comment:
Not sure about the react side, but on the vue side I get an error if it's pointed to localhost.
I seem to recall someone else running into this a while back. Not sure if there is a more proper fix.